src/Sulu/Bundle/WebsiteBundle/Tests/Unit/EventListener/SegmentCacheListenerTest.php

```diff
@@ -19,7 +19,7 @@
 use Symfony\Component\HttpFoundation\Request;
 use Symfony\Component\HttpFoundation\Response;
 
-class SegmentCacheListenerTests extends TestCase
+class SegmentCacheListenerTest extends TestCase
```

Comment:
test method changes should be done on 2.5 branch if possible and non breaking changes